README.md

```diff
@@ -254,3 +254,7 @@ Note: The [Patchwork](https://github.com/FlorentRevest/vscode-patchwork) and
 [Git Send Email](https://github.com/FlorentRevest/vscode-git-send-email)
 extensions are homegrown and maintained in other git repositories.
 Pull-requests or bug reports are also welcome there.
+
+## LPC Talk
```

Comment:
I'd prefer if this section was moved higer up in between the "Features" and "System-wide installation" sections